test/extensions/filters/http/stateful_session/config_test.cc

```diff
@@ -40,6 +40,10 @@ constexpr absl::string_view NotExistYaml = R"EOF(
     typed_config: {}
 )EOF";
 
+constexpr absl::string_view EmptyRouteYaml = R"EOF(
```

Comment:
Seems like its the session that is empty, not the route?

Reply:
Yeah, I will update it. Thanks.